pkg/util/instancemetadata/prod_test.go

```diff
@@ -126,6 +154,10 @@ func TestPopulateInstanceMetadata(t *testing.T) {
 			if p.resourceGroup != tt.wantResourceGroup {
 				t.Error(p.resourceGroup)
 			}
+
+			if !reflect.DeepEqual(p.environment, tt.wantEnvironment) {
```

Comment:
nit: pointer comparison is probably fine
```golang
			if p.environment != tt.wantEnvironment {
```

Reply:
Pointers won't be equal - see above

Reply:
You're right.  Bah.